### PR TITLE
HOST now also contains the protocol (https/http)

### PR DESCRIPTION
**Pull request by ulich**
introduced APPLICATION_URL and made HOST deprecated

### CODE REVIEW

File: lib/baustelle/elastic_beanstalk/environment.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    class EnvironmentConfiguration
+
+      def initialize(configuration_settings)
+        @options = configuration_settings.option_settings
+      end
+
+      def env_vars()
+        env = @options.select { |o| o.namespace == "aws:elasticbeanstalk:application:environment" }
+        env.each_with_object({}) { |opt,map| map[opt.option_name] = opt.value }
+      end
```
Comment (suxor42) [inline lib/baustelle/elastic_beanstalk/environment.rb]: Can we be consistent within a file?
Either always use o or always use opt as variable when iterating through the options.
I would prefer opt.

File: lib/baustelle/elastic_beanstalk/environment.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+module Baustelle
+  module ElasticBeanstalk
+    class EnvironmentConfiguration
```
Comment (sevos) [inline lib/baustelle/elastic_beanstalk/environment.rb]: class name should match the file name

File: lib/baustelle/elastic_beanstalk/environment.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+      def initialize(configuration_settings)
+        @options = configuration_settings.option_settings
+      end
+
+      def env_vars()
```
Comment (sevos) [inline lib/baustelle/elastic_beanstalk/environment.rb]: it's common to not leave empty brackets when there are no params

File: lib/baustelle/elastic_beanstalk/environment.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
+module Baustelle
+  module ElasticBeanstalk
+    class EnvironmentConfiguration
+
+      def initialize(configuration_settings)
+        @options = configuration_settings.option_settings
```
Comment (sevos) [inline lib/baustelle/elastic_beanstalk/environment.rb]: In this project, configuration is usually full or partial YAML structure (sometimes converted to Ruby objects).

In this case you seem to assert against cloud formation template, I'd consider renaming

File: lib/baustelle/script/cli.rb [hunk 5]
```diff
@@ -25,13 +29,17 @@ def wait_until_app(env_name, attribute, expected)
            "Prints the environment variables that can be injected in jenkins systemtests jobs"
       option "alternate-dns", desc: "Alternate dns name instead of using the <app>.elasticbeanstalk.com domain as HOST variable"
       def systemtests_env(app_name, env_name)
-        env = env_vars(app_name, env_name)
+        eb = Baustelle::ElasticBeanstalk::Client.new(region())
```
Comment (sevos) [inline lib/baustelle/script/cli.rb]: It's not common to use empty brackets in method calls

**Comment by sevos** [comment]
reviewed

**Pull request merged**